src/ripple/nodestore/README.md

```diff
@@ -33,19 +33,14 @@ contains. The fields are as follows:
 
  A 256-bit hash of the blob.
 
-* `mLedgerIndex`
-      
- An unsigned integer that uniquely identifies the ledger in which this 
-   NodeObject appears.
-
 * `mData`
       
  A blob containing the payload. Stored in the following format.
  
 |Byte   |                     |                          |
 |:------|:--------------------|:-------------------------|
-|0...3  |ledger index         |32-bit big endian integer |
-|4...7  |reserved ledger index|32-bit big endian integer |
+|0...3  |unused               |32-bit big endian integer |
```

Comment:
@nbougalis or @JoelKatz, do you have an opinion about whether marking these fields as unused is adequate?  It is possible that they are currently unused but, for example, must never be reused for any other purpose (with an appropriate explanation).  It's outside my area of expertise, but I thought I should ask the question.

Reply:
I'm OK with documenting as is "unused" and leaving eternity to unfold in a possible different direction.